src/CMakeLists.txt

```diff
@@ -102,7 +100,15 @@ else()
                     LIBS ${UMF_LIBS})
 endif()
 
-target_compile_definitions(umf PUBLIC ${UMF_COMPILE_DEFINITIONS})
+target_compile_definitions(umf PUBLIC ${UMF_PUBLIC_COMPILE_DEFINITIONS})
+
+if(UMF_BUILD_OS_MEMORY_PROVIDER AND WINDOWS)
+    # Disable the warning:
+    # hwloc-win64-build-2.10.0\include\hwloc/helper.h(1266):
+    # warning C4996: 'sscanf': This function or variable may be unsafe.
+    # Consider using sscanf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS.
+    target_compile_definitions(umf PRIVATE _CRT_SECURE_NO_WARNINGS)
```

Comment:
I think this is too strong because it disables all warning for the whole lib - I would prefer to use 

#ifdef windows
#pragma warning( push )
#pragma warning( disable : 4101)
#include <hwloc.h>
#pragma warning( pop )

Reply:
Right. Thanks!
Done.